insights/ui/login.py

```diff
@@ -0,0 +1,34 @@
+import configparser
+from selenium_utility import SeleniumUtility
+from locators import LoginPageLocator, InsightPageLocator
+
+
+class LoginUI(SeleniumUtility):
+    def __init__(self):
+        super(LoginUI, self).__init__()
+        config = configparser.ConfigParser()
+        config.read("pool.conf")
+        self.base_url = config.get('rhn_login', 'base_url')
+        self.username = config.get('rhn_login', 'rhn_username')
+        self.password = config.get('rhn_login', 'rhn_password')
+        self.driver.get(self.base_url)
+
+    def login_to_portal(self):
+        username_field = self.find_element(LoginPageLocator['username_field'])
+        username_field.send_keys(self.username)
+        password_field = self.find_element(LoginPageLocator['password_field'])
+        password_field.send_keys(self.password)
+        submit_btn = self.find_element(LoginPageLocator['submit_btn'])
+        submit_btn.click()
+        # Navigate to Insight_tab
+        current_url = self.driver.current_url
+        self.driver.get(current_url + 'insights/actions')
+        self.driver.implicitly_wait(30)
+
+    def register_system(self,hostname):
```

Comment:
what does this function do? add comments as to what the function does.
It looks like the function has nothing to with login but insights, why is this a part of loginUI class?